packages/console/src/pages/SignInExperience/utils/form.ts

```diff
@@ -116,9 +116,9 @@ export const getSignUpAndSignInErrorCount = (
 export const getOthersErrorCount = (
   errors: FieldErrorsImpl<DeepRequired<SignInExperienceForm>>
 ) => {
-  const { termsOfUse } = errors;
+  const { termsOfUseUrl } = errors;
```

Comment:
`(errors) => errors.termsOfUseUrl ? 1 : 0`

Reply:
btw what `getOthersError` suppose to cover? Only terms of use?

Reply:
Updated.
Yes, we only care `termsOfUrl` error on the `Others` tab, for other fields won't cause errors. E.g., select field, switch filed.